website/content/workbooks/workbook-5.md

```diff
@@ -0,0 +1,18 @@
++++
+title="CYF+01 Sprint 5"
+date="09 Jan 2023"    
+versions=["1-0-0"]
++++
+
+## Study
+
+- [ ] [Distributed Systems: Asynchronous Work and Pipelines](../../primers/distributed-software-systems-architecture/distributed-locking-and-distributed-consensus/)
+- [ ] [Troubleshooting Primer](../../primers/troubleshooting/)
+
+## Projects
+
+- [ ] [RAFT OTEL](../../projects/raft-otel) Do the raft-otel project. This large project explores a distributed consensus protocol using distributed tracing.
+
+## Product
+
+https://github.com/orgs/CodeYourFuture/projects/19/views/6?filterQuery=+-status%3ABacklog%2CDone+
```

Comment:
Link takes us to 404

Reply:
It's team locked, alas. Do you think we should put a note on about volunteering at CYF?

Reply:
I think it is a good idea.